### DISCUSSION TITLE
Location Information Not Updating

### DISCUSSION THREAD

**Discussion by ljmorrone** [code4recovery/12-step-meeting-list#561] [Q&A] [answered]
Hi,

The location information is not updating to "yes" for "Can I currently attend this meeting in person?" for two of our meetings. The two meetings are at the same location. This is not happening for other meetings at other locations. Every time I select "yes" and click "update", the selection reverts back to "no". 

This meeting https://albuquerqueaa.org/meetings/serenity-group-3/ shows as Temporarily Inactive when it is active. 

This meeting https://albuquerqueaa.org/meetings/progress-not-perfection-2/ shows as Online when it is both online and in-person.

I believe I am using plug-in version 3.12, and WordPress version 5.8.1.

Thank you

Edit: I am now having a similar issue with this meeting: https://albuquerqueaa.org/meetings/serendipity-group/, where I am trying to add a URL in the Meeting Information, but every time that I put in the URL and click 'update', the URL goes away and the meeting is not updated.

**Comment by tech2serve** [comment, marked as answer]
Hey @ljmorrone,

Please open up a debugging console, check "yes" then re-enter the address for the meeting as "601 Montano Rd NW, Albuquerque, NM". In the debug console, you should see a geocoded object that you can expand. Screen cap that, and post it here please. It should look similar to this, but it may say "cached"

<img width="563" alt="image" src="https://user-images.githubusercontent.com/31904316/140625335-5f6ed971-aeb5-452f-8b13-bad072039dc7.png">

If you see any errors in the console, please post those here too.

Please confirm you're entering these manually and not importing a CSV file?
Is this meeting part of a Group?

We've seen this behavior one other time if I recall correctly, and it turned out we had to override the address. However, this situation looks a little different, so I want to see what is in your cache.

If it ends up looking correct, then perhaps we'll have to look closer at your database. In general, if you select "yes" for attending in person, and the address is specific (i.e., not approximate because you've entered a complete street number and name), the meeting should publish without the TC type.

@code4recovery/twelve-step-meeting-list

**Reply by ljmorrone** [reply]
<img width="1149" alt="Screen Shot 2021-11-12 at 7 54 25 PM" src="https://user-images.githubusercontent.com/64061563/141603291-ba637c02-39ea-4fd6-9b31-8ac6f70e4b06.png">

I am entering these manually and not importing a CSV file.

This meeting is part of a Group (it is the only one in the Group)

**Reply by ljmorrone** [reply]
@tech2serve Is that enough information? Please let me know if I need to provide anything else. I am now having an issue with this meeting: https://albuquerqueaa.org/meetings/serendipity-group/, where I am trying to add a URL in the Meeting Information, but every time that I put in the URL and click 'update', the URL goes away and the meeting is not updated.

**Reply by ljmorrone** [reply]
@tech2serve Additionally, I have noticed that all three meetings with this type of issue are located on Montaño Rd NW in Albuquerque. Not the same location, but the same street. When I create a new meeting listing with the same information, the problem persists. Any insight? Some groups are getting a little frustrated with their meeting listing.

**Reply by tech2serve** [reply]
@ljmorrone,

Let me apologize for one of us not getting back to you sooner. I have no excuse.

Since there appear to be a couple of issues going on here, it would help me if you could include in your comment to this thread some screen caps from your edit meeting page. For example, where are you trying to add a URL? 

Also, you can export your meeting database to a csv file ( go to Meetings-->Import and Settings, then look on the right side of the screen where is says where to download) which you can then email to me at devs@code4recovery.org. I'll then take a closer look at the data.
<img width="273" alt="Untitled" src="https://user-images.githubusercontent.com/31904316/146654560-aa9f1ef3-2b7b-4f4a-a17c-2b0af1aca731.png">

**Reply by ljmorrone** [reply]
@tech2serve Thanks for getting back to me! Here are some screen caps for the Serenity group situation
<img width="1034" alt="Screen Shot 2021-12-18 at 2 41 41 PM" src="https://user-images.githubusercontent.com/64061563/146656209-1b0498c7-f6b7-4307-9954-366d66864408.png">
I select "yes" for location open

<img width="289" alt="Screen Shot 2021-12-18 at 2 41 55 PM" src="https://user-images.githubusercontent.com/64061563/146656216-aab0afcc-41e8-4b7a-97a5-ece8f922ec70.png">
I click "update"
<img width="450" alt="Screen Shot 2021-12-18 at 2 42 07 PM" src="https://user-images.githubusercontent.com/64061563/146656217-a9257a30-c7a1-4c64-921b-8e570f427d1c.png">
It says post has been updated and I click on "view post"
<img width="1255" alt="Screen Shot 2021-12-18 at 2 42 19 PM" src="https://user-images.githubusercontent.com/64061563/146656220-a055fc94-cad0-4eed-89a9-24993a703ebf.png">
Then it still says "temporarily inactive"

**Reply by ljmorrone** [reply]
Here is where I try to put the URL for the Zoom meeting for the Serendipity group: 
<img width="961" alt="Screen Shot 2021-12-18 at 2 45 55 PM" src="https://user-images.githubusercontent.com/64061563/146656267-907c3439-e2fc-493c-b3f0-799b4649915b.png">

And the join Zoom meeting button/online designation will not appear after I update

**Reply by tech2serve** [reply]
I’ll experiment with the CSV and get back to you.

In the meantime, is this the only meeting for which you’re having trouble with the Zoom URL?

If I’m understanding correctly, this meeting should show as a hybrid attendance option. Is that what you’re expecting?

**Reply by ljmorrone** [reply]
Thank you! Yes, that is the only meeting for which I am having trouble with the Zoom URL. It is both online and in-person, but only shows as an in-person meeting. And for these two meetings: This meeting https://albuquerqueaa.org/meetings/serenity-group-3/ and https://albuquerqueaa.org/meetings/progress-not-perfection-2/ I am having trouble getting them to show as in-person (one is also online and one is not, and their online status is accurate). Essentially, I have had trouble with any and all changes that I have attempted to make to a meeting that has a location on Montaño Road NW in Albuquerque since my original post.

**Reply by tech2serve** [reply]
Taking one at a time here, starting with `serenity-group-3`, I imported the csv and got this:

<img width="516" alt="image" src="https://user-images.githubusercontent.com/31904316/146658796-97e50427-a976-4abc-b586-04429ea1e224.png">

Have you cleared your server cache? If not, let's try that. I'm also wondering if you've tried disabling your other plugins and checking if the problem persists?

Do you have a test site you can set up a fresh install? 

If all of these don't clear the issue, we'll need to dive into your database and explore your local cache. But first let me know about the other ideas.

**Reply by ljmorrone** [reply]
I have not cleared our server cache because I do not know where our server is. I will have to ask about that at our Central Office meeting next week. I did just deactivate all other plugins, and the problem did persist.

**Reply by Grateful-Viv** [reply]
For own site hosted by SiteGround, we use SiteGround Optimizer plugin, but there are also a number of other plugins you can try. Another site I maintain uses Comet Cache. This is by no means an endorsement of one over the other, but it may help, even if it doesn't resolve the particular problem you are experiencing with TSML.

**Reply by ljmorrone** [reply]
The cache is clear. The problem persists on our site. However, on our devsite, I imported the csv, and when importing deleted all meetings, locations, groups, districts, and regions. This resolved the issue for serenity group. We got the same thing in the screenshot you posted when you imported the csv. This did not resolved the issue with the other two meetings, unfortunately.

**Reply by tech2serve** [reply]
For the Serendipity Group meeting on my dev site, I just added a Zoom URL and it published just fine. Pleas share with me _exactly_ what you're entering for a URL. Perhaps a screen cap right before you click on publish. 

For Progress Not Perfection, I'm showing it on your live site as being online and in-person. Is that not what you want or are seeing?

**Reply by ljmorrone** [reply]
Progress Not Perfection is now showing up correctly on the live site after I imported the csv on the live site in the same way I did on our dev site. It was not showing up like that on the devsite, but now it is resolved!

**Reply by ljmorrone** [reply]
https://hp.zoom.com/j/8141644981?pwd=anI3Z3gwRWZlaDl0Mm8ycnBLcGZldz09 that is the link that I was sent from Serendipity Group

**Reply by tech2serve** [reply]
The correct TLD (top level domain) for Zoom is `.us`. Please change`.com` to `.us` in the URL, and I believe you'll find it works.

**Reply by ljmorrone** [reply]
That works! Thank you!!!
